doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -136,6 +149,11 @@ The pooling data source configuration relies on additional resource that have be
 . Configure {KIE_SERVER} to use the data source:
 .. Open one of the following scripts in a text editor:
 +
+[NOTE]
+====
+The `setenv.sh` or `setenv.bat` script should already exist. However, if it does not, create it.
```

Comment:
Why (what circumstances) would it not exist?

Reply:
Excellent question. I'll ask an SME.